docs/doc/30-reference/20-functions/10-aggregate-functions/aggregate-retention.md

```diff
@@ -36,21 +36,21 @@ INSERT INTO retention_test SELECT '2018-08-08', number FROM numbers(60);
 ```
 
 ```
-SELECT sum(r[0]) as r1, sum(r[1]) as r2 FROM (SELECT uid, retention(date = '2018-08-06', date = '2018-08-07') AS r FROM retention_test WHERE date = '2018-08-06' or date = '2018-08-07' GROUP BY uid);
+SELECT sum(r[1]) as r1, sum(r[2]) as r2 FROM (SELECT uid, retention(date = '2018-08-06', date = '2018-08-07') AS r FROM retention_test WHERE date = '2018-08-06' or date = '2018-08-07' GROUP BY uid);
```

Comment:
This modification does affect compatibility, do we need to notify users of this modification in advance?

Reply:
databend.rs who used retention also needs modified